inlong-sort/sort-flink/sort-flink-v1.15/sort-connectors/mysql-cdc/pom.xml

```diff
@@ -169,6 +169,10 @@
                                     <pattern>com.zaxxer</pattern>
                                     <shadedPattern>com.ververica.cdc.connectors.shaded.com.zaxxer</shadedPattern>
                                 </relocation>
+                                <relocation>
+                                    <pattern>io.debezium</pattern>
+                                    <shadedPattern>com.ververica.cdc.connectors.shaded.io.debezium</shadedPattern>
```

Comment:
Why shade dependencies with the `com.ververica.cdc.connectors` prefix?

Reply:
1、we copied mysql connector from ververica and their pattern is com.ververica.cdc.connectors, to keep consistent with their format
2、the rule is that if shaded class is from inlong we shade with inlong prefix, if not, shade with ververica prefix